Documentation/Contributing/ExperimentalFeatures.md

```diff
@@ -31,11 +29,35 @@ if your component is part of solvers at `Microsoft.MixedReality.Toolkit.Utilitie
 
 See [this PR](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4532) for an example. 
 
-### Graduating experimental code into MRTK code  
+### Minimize impact to MRTK code
+While your MRTK change might get your experiment to work, it could impact other people in ways you do not expect.
+Any regressions you make to the MRTK core code would result in your pull request getting reverted. 
+
+Aim to have zero impact on MRTK core code. 
+
+If you do have impact on MRTK code, write tests to ensure that your changes do not regress in the future.
+
+### Using you experimental feature should not impact people's ability to use core controls
```

Comment:
These titles seem exceeding long. => "Experimental features should not impact core controls" Gives same message with much less words, especially since there is a description below